Poker.BE/Poker.BE.DomainTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Poker.BE.DomainTests")]
```

Comment:
WHAT!? why `Poker.BE.DomainTests` had returned? i thought you got rid of it... please clean that